Rework of Home Page to Develop

Miramoop opened the pull request.

Commit:
Updates to Home

Commit:
Update Homepage Button Colors

Commit:
Merge branch 'develop' into feature/rework-of-home-page

Commit:
Update tailwind config & fix styling errors

Commit:
New Home Styles

Commit:
Removing Comments

Commit:
Remove empty  lines

Miramoop committed:
Merge branch 'develop' into feature/rework-of-home-page

tstirtz-bzs commented on src/components/Home.js:
```diff
@@ -2,18 +2,26 @@ import React from "react";
-    >
-      Start Quiz
-    </button>
+    <div className="hero" style={{ minHeight: "75vh" }}>
```
Does this min-height rule have to be inline? If not, it should be moved to a style file.

tstirtz-bzs replied:
Or you could extend tailwind

CStamBizStream commented on src/styles/home.scss:
```diff
@@ -2,15 +2,13 @@
   text-align: center;
   width: 100%;
   color: $mainTextColor;
+  background-image: linear-gradient(180deg, #315A42, #5A5731);
```
Looks like you have these values in your color variables. These could be cleaned up to use the variables instead

CStamBizStream commented on src/components/Home.js:
```diff
@@ -2,18 +2,26 @@ import React from "react";
-    >
-      Start Quiz
-    </button>
+    <div className="hero" style={{ minHeight: "75vh" }}>
```
Add inline style to css file instead

CStamBizStream commented on src/styles/home.scss:
```diff
@@ -2,15 +2,13 @@
   align-items: center;
-  background: $buttonLinkColor;
+  outline: 2px solid black;
+  background: #31355A;
```
Same thing here, it looks like you have color variables for this

CStamBizStream commented on src/styles/styles.css:
```diff
@@ -120,26 +120,28 @@ footer .copyright {
   text-align: center;
   width: 100%;
   color: #000;
+  background-image: linear-gradient(180deg, #315A42, #5A5731);
```
Use color variables

CStamBizStream commented on src/styles/styles.css:
```diff
@@ -120,26 +120,28 @@ footer .copyright {
   align-items: center;
-  background: #fff;
+  outline: 2px solid black;
+  background: #31355A;
```
Use Color Variables

Commit:
Pr #16 Fixes

Using correct style variables throughout sass file
Moving inline styles to sass file
